spec/factories/users.rb

```diff
@@ -14,6 +15,7 @@
       email 'aliceuser@example.com'
     end
 
+    # TODO: Bad factory, we are already seeding the test database with this user? Potential conflict with email being taken already
```

Comment:
We have a seeds file that has this user in already. Could be potential issue if we weren't wiping the database on every test.